app/invocation/invocation_execution_card.tsx

```diff
@@ -264,14 +272,22 @@ export default class ExecutionCardComponent extends React.Component {
       }
     }
 
-    const filteredActions = this.state.executions.filter(
-      (action) =>
-        !this.props.filter ||
-        `${action.actionDigest.hash}/${action.actionDigest.sizeBytes}`
-          .toLowerCase()
-          .includes(this.props.filter.toLowerCase()) ||
-        action.commandSnippet.toLowerCase().includes(this.props.filter.toLowerCase())
-    );
+    const filteredActions = this.state.executions
+      .filter(
+        (action) =>
+          !this.props.filter ||
+          `${action.actionDigest.hash}/${action.actionDigest.sizeBytes}`
+            .toLowerCase()
+            .includes(this.props.filter.toLowerCase()) ||
+          action.commandSnippet.toLowerCase().includes(this.props.filter.toLowerCase())
+      )
+      .filter(
+        (action) =>
+          this.state.statusFilter.includes("all") ||
```

Comment:
You can use `===` here instead of `includes` for `this.state.statusFilter`, here and below